hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/export/AlertDefinitions.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.export;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import java.util.List;
+
+import org.hawkular.alerts.api.model.action.ActionDefinition;
+import org.hawkular.alerts.api.model.trigger.FullTrigger;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Representation of a list of full triggers (trigger, dampenings and conditions) and actions definitions.
+ *
+ * @author Jay Shaughnessy
+ * @author Lucas Ponce
+ */
+public class AlertDefinitions {
```

Comment:
FullTriggers don't include ActionDefinitions IMO.
So a concept of FullTriggers + ActionDefinitions is needed.
AlertDefinitions came to me as the most direct definition.
But I can be open to negotiate :).